pertpy/tools/_metadata.py

```diff
@@ -42,7 +42,7 @@ def __init__(self):
         self.driver_gene_intOGen = pd.read_table(driver_gene_intOGen_file_path)
 
         """Download meta data for driver genes of the COSMIC Tier 1 gene from DepMap_Sanger """
-        # self.driver_gene_cosmic = pd.read_csv("Census_allSun Feb 12 10_56_28 2023.csv")
+        self.driver_gene_cosmic = pd.read_csv("COSMIC_tier1.csv")
```

Comment:
We might need to update the path when people are installing the package into their environments. We can also upload this to figshare and then download it lazily.

Reply:
@wxicu should it be on figshare?